docs/content/en/contributing/how-to-write-a-parser.md

```diff
@@ -294,12 +294,24 @@ This local command will launch the unit test for your new parser
 $ docker compose exec uwsgi bash -c 'python manage.py test unittests.tools.<your_unittest_py_file>.<main_class_name> -v2'
 {{< /highlight >}}
 
+or like this:
+
+{{< highlight bash >}}
+$ dc-unittest.sh --profile <profile> --test-case unittests.tools.<your_unittest_py_file>.<main_class_name>
+{{< /highlight >}}
+
 Example for the blackduck hub parser:
 
 {{< highlight bash >}}
 $ docker compose exec uwsgi bash -c 'python manage.py test unittests.tools.test_blackduck_csv_parser.TestBlackduckHubParser -v2'
 {{< /highlight >}}
 
+or like this:
+
+{{< highlight bash >}}
+$ dc-unittest.sh --profile postgres-rabbitmq --test-case unittests.tools.test_blackduck_csv_parser.TestBlackduckHubParser
```

Comment:
Profiles are not used anymore. They are automatically ignored. It might be good idea to remove them.